tests/testsuite.src/listings.at

```diff
@@ -97,9 +97,7 @@ AT_DATA([prog.cob], [
            STOP RUN.
 ])
 
-AT_CHECK([$COMPILE_ONLY -t prog.lst -tlines=0 prog.cob], [0], [], [])
-
-AT_DATA([prog1.lst],
+AT_CHECK([$COMPILE_LISTING -t- prog.cob], [0],
```

Comment:
I like that very much and given that cobc normally outputs to stderr this is also nicely to handle